functions-app/package.json

```diff
@@ -25,6 +25,7 @@
     "mssql": "^5.1.0",
     "object-hash": "^1.3.1",
     "ramda": "^0.26.1",
+    "ramda-adjunct": "^2.20.0",
```

Comment:
do we really need this package? methods used can be replaced with vanilla ones?

Reply:
I suspect the answer, at the moment, is that we could leave it behind. And yet, it offers such a wide range of useful ancillary functions to Ramda that it seems logical to include the library, especially as we get such great use from ramda for the processing piece.

We have many more detections to add, these will most likely make use of ramda so I want to include at this time.  Please note that ramda-adjunct does not have any dependencies other than ramda.

Reply:
Ok then it does make sense to keep it.